bika/lims/locales/es/LC_MESSAGES/bika.po

```diff
@@ -5496,7 +5526,7 @@ msgstr "Las muestras de este tipo deben ser tratadas como peligrosas"
 
 #: bika/lims/browser/dashboard/dashboard.py:658
 msgid "Samples received"
-msgstr ""
+msgstr "Muestras recepcionadas"
```

Comment:
@xispa same as the previous comment.

 _recibidas_ sounds more natural to me, but maybe the technical word is _recepcionadas_ so I am ok with whatever you prefer.